src/Components/Facility/FacilityUsers.tsx

```diff
@@ -117,13 +101,15 @@ export default function FacilityUsers(props: any) {
       return;
     }
     setIsFacilityLoading(true);
```

Comment:
You should be making use of the useQuery's loading state instead which is auto handled.